### PR TITLE
added TreeViewDemo

### PR DESCRIPTION
**Pull request by mhrimaz**
Added TreeView Demo & fixed typos.
last pull request closed because i messed up with git :/
https://github.com/JonathanGiles/fx-ui-controls/pull/10

### CODE REVIEW

File: src/main/java/org/modernclients/ch3/samples/TreeViewDemo.java [hunk 1]
```diff
@@ -1,14 +1,41 @@
...
+        TreeItem<String> treeItem22 = new TreeItem<>("CheckBox");
+        treeItem2.getChildren().addAll(treeItem21,treeItem22);
+
+        rootItem.getChildren().addAll(treeItem1,treeItem2);
+
+        TreeView<String> tree = new TreeView<>(rootItem);
+        tree.setOnMouseClicked(e->{
+            ObservableList<TreeItem<String>> selectedItems = tree.getSelectionModel().getSelectedItems();
+            console.accept("Selected item->"+selectedItems);
+        });
```
Comment (JonathanGiles) [inline src/main/java/org/modernclients/ch3/samples/TreeViewDemo.java]: don't use `tree.setOnMouseClicked()`, you should use `tree.getSelectionModel().getSelectedItems().addListener()`

**Review by JonathanGiles — CHANGES_REQUESTED** [review]
Just one thing that needs fixing. I can do if you want.

File: src/main/java/org/modernclients/ch3/samples/TreeViewDemo.java (hunk 1 above)
Comment (mhrimaz) [reply src/main/java/org/modernclients/ch3/samples/TreeViewDemo.java]: I wrote it the way you mentioned. I hope that it's the correct way

**Comment by JonathanGiles** [comment]
I will merge and tweak. Thanks!

**Pull request merged**